src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -3,55 +3,224 @@
+.qps-container .query-plan .properties {
+	flex: 0 0 500px;
+	overflow: hidden;
+	width: 500px;
+	height: 100%;
+	border: 1px solid;
+	display: flex;
+	flex-direction: column;
+	border: 1px solid;
+}
+
+/* Title container of the properties view */
+.qps-container .query-plan .properties .title {
+	line-height: 22px;
+	height: 22px;
+	font-size: 11px;
+	font-weight: bold;
+	text-transform: uppercase;
+	overflow: hidden;
+	display: flex;
+	align-items: center;
+	box-sizing: border-box;
+	margin-top: 3px;
+	margin-bottom: 5px;
+	padding-left: 5px;
+	border-bottom: solid 1px;
+	flex: 0 0 25px;
+}
+
+/* text in title container of properties view */
+.qps-container .query-plan .properties .title .text {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	font-size: 11px;
+	min-width: 3ch;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
+	flex: 1;
+}
+
+/* action bar in the title container for the properties view. This contains the close icon */
+.qps-container .query-plan .properties .title .action-bar {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	font-size: 11px;
+	min-width: 3ch;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
+	flex: 0 0 30px;
+}
+
+/* Operation name styling in the properties view. */
+.qps-container .query-plan .properties .operation-name {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	font-size: 13px;
+	min-width: 3ch;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
+	flex: 0 0 25px;
+	margin-top: 3px;
+	margin-bottom: 5px;
+	margin-left: 5px;
+}
+
+/* Properties table container in properties view */
+.qps-container .query-plan .properties .table-container {
 	overflow-y: scroll;
-	background-color: #eeeeee;
-	width: 510px;
+	flex: 1;
+	flex-grow: 1;
+	border-top: 1px solid;
+}
+
+/* Action bar for the query plan */
+.qps-container .query-plan .action-bar-container {
+	flex: 0 0 25px;
 }
 
-.qp-container .prop-table-header {
+/* styling for the column headers in the properties table */
+.qps-container .properties .table-container .prop-table-header {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	font-size: 11px;
+	min-width: 3ch;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
 	font-weight: bold;
+	text-transform: uppercase;
 }
 
-.qp-container .properties-header {
+.qps-container .properties-header {
 	font-weight: bold;
 	padding: 5px;
 }
 
-.qp-container .properties-toggle {
+.qps-container .properties-toggle {
 	height: 9px;
 	width: 9px;
 	display: inline-block;
 }
 
-.qp-container .properties-toggle .expand {
+.qps-container .properties-toggle .expand {
 	background: url(../images/expand.gif) no-repeat center center;
 }
 
-.qp-container .properties-toggle .collapse {
+.qps-container .properties-toggle .collapse {
 	background: url(../images/collapse.gif) no-repeat center center;
 }
 
-.qp-container .mxTooltip {
+.qps-container .mxTooltip {
 	-webkit-box-shadow: 3px 3px 12px #C0C0C0;
```

Comment:
hard coded colors

Reply:
These stylings need to be fixed from azdata graph library. I will make those fixes in a future PR